src/operation.ts

```diff
@@ -254,27 +305,24 @@ class Operation {
   /**
    * Return an array of all tags, and their metadata, that exist on this operation.
    *
-   * @returns {array}
    */
-  getTags() {
+  getTags(): Array<RMOAS.TagObject> {
     if (!('tags' in this.schema)) {
       return [];
     }
 
-    let oasTags = new Map();
+    const oasTags = new Map();
     if ('tags' in this.api) {
       this.api.tags.forEach(tag => {
         oasTags.set(tag.name, tag);
       });
     }
 
-    oasTags = Object.fromEntries(oasTags);
```

Comment:
Removed this nonsense to just use `Map.has()` and `Map.get()` instead like a non-galaxy brained person.

<sub>what the heck was i thinking</sub>